core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponentSpec.scala

```diff
@@ -10,17 +10,16 @@ import org.broadinstitute.dsde.rawls.model._
 class MethodConfigurationComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers with RawlsTestUtils {
   import driver.api._
 
-  "MethodConfigurationComponenent" should "save and get a method config" in withDefaultTestDatabase {
+  "MethodConfigurationComponent" should "save and get a method config" in withDefaultTestDatabase {
     val workspaceContext = SlickWorkspaceContext(testData.workspace)
 
     val methodConfig2 = MethodConfiguration(
       "ns",
       "config2",
       "sample",
-
-      Map("input.expression" -> AttributeString("this..wont.parse")),
-      Map("output.expression" -> AttributeString("output.expr")),
       Map("prereq.expression" -> AttributeString("prereq.expr")),
+      Map("input.expression" -> AttributeString("this.input_expr")),
```

Comment:
This change is not strictly necessary - it originally came from my local 2444 wip branch